### ISSUE TITLE
Document how to contribute

### ISSUE THREAD

**Issue by JMLX42** [democratech/2022#7]

**Comment by jerome-caucat** [comment]
I agree, we could add a `How to contribute` section in the README or in a new CONTRIBUTING.md file.

Here are some resources we can link:
- https://opensource.guide/how-to-contribute/#how-to-submit-a-contribution
- https://egghead.io/lessons/javascript-exploring-a-repository

For information here is what I did:
- Sign up/in to https://github.com.
- Fork this repository.
- Install/setup git on my machine.
- Clone my own fork of the repository.
- Proceed with the `Development` instructions from the [README](https://github.com/democratech/2022#1-development).

Next steps should then be:
- Find an issue where I can contribute.
- Comment in this issue:
    - To make sure the feature or bug-fix is still wanted.
    - To ask if and how I can contribute and offer my help.
- Make changes in a new branch on my own fork of the repository.
- Create a Pull Request.

@JMLX42 Is that how it should be done?

**Comment by JMLX42** [comment]
@jerome-caucat yes. But we must also document the issue labels, how they are used, and the workflow to submit/review PRs.
